pyogrio/tests/test_geopandas_io.py

```diff
@@ -100,6 +100,21 @@ def test_read_null_values(test_fgdb_vsi):
     assert df.loc[df.SEGMENT_NAME.isnull()].SEGMENT_NAME.iloc[0] == None
 
 
+def test_read_set_fid_index(naturalearth_lowres):
+    kwargs = {"skip_features": 2, "max_features": 2}
+
+    # default is to not set FIDs as index
+    df = read_dataframe(naturalearth_lowres, **kwargs)
+    assert np.array_equal(df.index.values, pd.RangeIndex(0, 2).values)
```

Comment:
```suggestion
    assert_index_equal(df.index, pd.RangeIndex(0, 2)
```
I think it is better to use dedicated `assert_index_equal` here. It also needs to be imported above 

```py
from pandas.testing import assert_index_equal
```

Same below and we can even skip `assert df.index.name == "fid"` here because that gets tested implicitly.